cmd/operator-sdk/test/local.go

```diff
@@ -50,10 +50,11 @@ type testLocalConfig struct {
 	namespace          string
 	upLocal            bool
 	noSetup            bool
-	debug              bool
 	image              string
 	localOperatorFlags string
+        debug              bool
```

Comment:
All bool vars need to be declared at the end and strings on top. 
Also, please run go ftm ./... to fix the indentation,